### PR TITLE
update actix-http/README.md

### PR DESCRIPTION
**Pull request by nwtgck**
Hi! This is very tiny update of actix-http/README.md. (see: Files changed https://github.com/actix/actix-web/pull/1700/files)

## PR Type

Other

## PR Checklist
Check your PR fulfills the following:

- [ ] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [ ] A changelog entry has been made for the appropriate packages.
- [ ] Format code with the latest stable rustfmt

### CODE REVIEW

File: actix-http/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Actix http
 ## Example
 
 ```rust
-// see examples/framed_hello.rs for complete list of used crates.
+// see examples/hello-world.rs for complete list of used crates.
```
Comment (popzxc) [inline actix-http/README.md:16]: This code snippet currently looks exactly the same as the linked example. Do we need this comment at all?

Comment (nwtgck) [reply actix-http/README.md:16]: @popzxc I have the same idea.

The only difference is as follows. The example uses `use futures_util::future`.

```diff
- use futures::future;
+ use futures_util::future;
```

https://github.com/actix/actix-web/blob/1596893ef7275a2c6530e09abaec7e2cc2f0bf2c/actix-http/examples/hello-world.rs#L5

Comment (JohnTitor) [reply actix-http/README.md:16]: It has the same effect, I wouldn't mind either way.

Comment (nwtgck) [reply actix-http/README.md:16]: @JohnTitor 

Thanks. The Cargo.toml depends `futures-util`, not `futures`. The new users of actix-http like me can easily to copy the line ` futures-util = ...` from the Cargo.toml and past.
https://github.com/actix/actix-web/blob/1596893ef7275a2c6530e09abaec7e2cc2f0bf2c/actix-http/Cargo.toml#L62

Comment (JohnTitor) [reply actix-http/README.md:16]: I mean, it has the same effect so we could just tweak the example and remove that comment on README.

Comment (nwtgck) [reply actix-http/README.md:16]: I think I should close this PR and someone maintained the README can edit/remove README.md as they like.

**Pull request closed without merging**